cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/protocol/oauth/oAuthFailureView.jsp

```diff
@@ -19,9 +19,13 @@
 
 --%>
 <jsp:directive.include file="../../default/ui/includes/top.jsp" />
+
 <div id="msg" class="errors">
     <h2><spring:message code="screen.oauth.error.header" /></h2>
-    <p><spring:message code="${rootCauseException.message}" /></p>
     <p><spring:message code="${rootCauseException.code}" /></p>
+    <p><spring:message code="${rootCauseException.message}" /></p>
```

Comment:
Here is an example of CAS OAuth failure page:
![cas-oauth-failure](https://cloud.githubusercontent.com/assets/3750414/20847301/73d8ab08-b89b-11e6-874a-8233bd439980.png)